libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboardStyles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
as Ilya mentioned, style files should be in format xyz.styles.ts. Any specific reason why it wasn't updated?